chroma-manager/tests/integration/existing_filesystem_configuration/utils/create_lustre_filesystem.py

```diff
@@ -107,8 +107,8 @@ def _clear_current_target_devices(self):
 
         for server in config['lustre_servers']:
             self.remote_command(server['address'],
-                                'partprobe; udevadm settle',
-                                expected_return_code=None)
+                                'reboot',
```

Comment:
It seems drastic to need a reboot just to remedy the MGT not mounting.  What kind of failure was happening on mount?